libs/designer-ui/src/lib/chatbot/components/chatBubble.tsx

```diff
@@ -71,6 +79,15 @@ export const ChatBubble: React.FC<ChatBubbleProps> = ({
             <div className={'msla-bubble-footer-disclaimer'}>{intlText.aIGeneratedDisclaimer}</div>
             {onThumbsReactionClicked && (
               <div className={'msla-bubble-reactions'}>
+                <IconButton
+                  className={'msla-copy-button'}
+                  title={'Copy'}
+                  iconProps={{ iconName: 'Copy' }}
+                  onClick={() => {
+                    const text = reactElementToText(children);
```

Comment:
instead of extracting the string from the Markdown children props, let's just pass down the text down to the chatBubble. This way we can also conditionally render this copy button whether or not there is a text there - currently this would happen in all cases of the chat bubble

Reply:
I made those changes, when you get a chance please review the latest code!

Reply:
Made the changes! please take a look at the latest when you get a chance and let me know if there is anything else!